nltk/classify/api.py

```diff
@@ -38,12 +43,13 @@ class ClassifierI(object):
     Subclasses may define:
       - either ``prob_classify()`` or ``prob_classify_many()`` (or both)
     """
+    @abstractmethod
     def labels(self):
         """
         :return: the list of category labels used by this classifier.
         :rtype: list of (immutable)
         """
-        raise NotImplementedError()
+        pass
```

Comment:
@dimazest so is it better to remove the `pass` statements? Most examples of abstract methods I've seen elsewhere leave it in. I think it's a bit more explicit and clear.